### DISCUSSION TITLE
Fotovoltaico SoleEdge Private API

### DISCUSSION THREAD

**Discussion by Koaz74** [indomus/forum#65] [HUB: Home Assistant]
Buon pomeriggio,

Obiettivo: ho un fotovoltaico con inverter Solaredge e vorrei aumentare l'autoconsumo attraverso la domotica.

Premessa: sto utilizzando contemporaneamente Homey e Hoobs ma entrambi prevedono solo l'utilizzo dell' API Key standard con interrogazioni limitate a 300 al giorno.

Ho provato con successo a configurare OpenHub per quanto attiene la problematica in questione ma purtroppo mancano in questo hub alcune compatibilità fondamentali con altra domotica da me utilizzata.

 Domanda:qualcuno ha esperienza in Home Assistant sull' utilizzo della Private API anziché la API Key per quanto riguarda il superamento delle 300 interrogazioni giornaliere nel sistema di monitoraggio SolarEdge?

Grazie

**Comment by gabrio79** [comment]
hai dato un occhiata qui?
https://community.openhab.org/t/reading-data-from-solaredge-inverters-via-modbus-tcp/26290

**Reply by Koaz74** [reply]
Grazie per la dritta

**Reply by fizzi** [reply]
Se il tuo inverter ha una interfaccia modbus puoi accedere a tutti parametri ...

**Reply by Koaz74** [reply]
interfaccia modbus presente, stasera abilito il tcp nei parametri

**Reply by fizzi** [reply]
Ottimo ... recupera i registri che ti interessano e vai a bomba... 👍

**Reply by nite88** [reply]
Ciao ragazzi, ne approfitto per chiedervi una mano per la configurazione modbus, con home assistant, di un sistema di fancoil. Ho i registri forniti dal produttore, e ho provato la guida presente sul sito, ma non riesco a far comunicare home assistant con i vari fancoil. Per interfacciare la Lan con la rete modbus ho preso un dispositivo della usr-dr302.
